dss/storage/bundles.py

```diff
@@ -22,44 +22,66 @@
     f"^{BUNDLE_PREFIX}/({UUID_PATTERN})(?:\.(" + VERSION_PATTERN + "))?\." + TOMBSTONE_SUFFIX + "$")
 # matches all bundle objects
 DSS_OBJECT_NAME_REGEX = re.compile(
-    f"^{BUNDLE_PREFIX}/({UUID_PATTERN})(?:\.({VERSION_PATTERN}))?(?:\.{TOMBSTONE_SUFFIX})?$")
+    f"^({BUNDLE_PREFIX}|{FILE_PREFIX})/({UUID_PATTERN})(?:\.({VERSION_PATTERN}))?(\.{TOMBSTONE_SUFFIX})?$")
 
 
-def bundle_key_to_bundle_fqid(bundle_key: str) -> str:
-    match = DSS_OBJECT_NAME_REGEX.match(bundle_key)
-    uuid, version = match.groups() if match else (None, None)
-    if uuid and version:
-        return format_bundle_fqid(uuid, version)
-    else:
-        raise ValueError(f"Object name does not contain a valid bundle identifier: {bundle_key}")
+class ObjectIdentifier(namedtuple('ObjectIdentifier', 'uuid version')):
 
+    @classmethod
+    def from_key(cls, bundle_key: str):
```

Comment:
If this a generic `ObjectIdentifier` then `bundle_key` can be any key, not just a bundle key correct?

Reply:
✅